python_scripts/ensemble_hyperparameters.py

```diff
@@ -142,12 +149,6 @@
 cv_results[columns].sort_values(by="mean_test_error")
 
 # %% [markdown]
-#
-# ```{caution}
-# Here, we tune the `n_estimators` but be aware that using early-stopping as
-# in the previous exercise will be better.
-# ```
-#
```

Comment:
I think we should keep this note here as using early stopping for `GradientBoosting*` models is often a good idea but grid-searching `n_estimators` is an ok-ish alternative to me.